app-misc/zellij/zellij-0.40.1-r1.ebuild

```diff
@@ -562,9 +565,11 @@ src_compile() {
 	cargo_src_compile
 	popd 2>/dev/null || die
 
-	if use doc; then
-		lowdown -t man -o ${PN}.1 docs/MANPAGE.md || die
-	fi
+	go-md2man -in docs/MANPAGE.md -out ${PN}.1 || die
+
+	target/$(usex debug debug release)/zellij setup --generate-completion bash > zellij.bash || die
+	target/$(usex debug debug release)/zellij setup --generate-completion fish > zellij.fish || die
+	target/$(usex debug debug release)/zellij setup --generate-completion zsh > zellij.zsh || die
```

Comment:
Replacing `target/$(usex debug debug release)` with `"$(cargo_target_dir)"` future-proofs the path in case cargo.eclass sets the target ABI unconditionally in the future.

The commit message for 73aef61ba818739034dd5cc939a315213bc77741 explains this quite nicely, even though the cross-compiling argument doesn't apply here.

Reply:
Thank you